roles/agent/README.md

```diff
@@ -4,11 +4,33 @@ Install and configure a [Beszel](https://github.com/henrygd/beszel) binary agent
 
 ## Role Variables
 
+### Authentication Methods
+
+Beszel supports two authentication methods:
```

Comment:
Could you please alter the wording slightly to specifically mention this is authentication methods for agents?

> Beszel supports two authentication methods for agents: